### PR TITLE
MWPW-145669: disabled IMS profile matching fields

### PR DESCRIPTION
**Pull request by yeibercano**
Describe your specific features or fixes

Resolves: [MWPW-145669](https://jira.corp.adobe.com/browse/MWPW-145669)

Test URLs:
- Before: https://main--events-milo--adobecom.hlx.page/
- After: https://mwpw-145669--events-milo--adobecom.hlx.page/drafts/cano/details-page#rsvp-form

### CODE REVIEW

**Comment by yeibercano** [comment]
https://github.com/adobecom/events-milo/assets/15808161/0ffd6764-e545-49a5-a72b-0e1d7728113e

File: blocks/events-form/events-form.js [hunk 1]
```diff
@@ -408,7 +408,10 @@ function personalizeForm(form, resp) {
 
   Object.entries(resp).forEach(([key, value]) => {
     const matchedInput = form.querySelector(`#${snakeToCamel(key)}`);
-    if (matchedInput) matchedInput.value = value;
+    if (matchedInput) {
+      matchedInput.value = value;
+      matchedInput.disabled = "true";
```
Comment (qiyundai) [inline blocks/events-form/events-form.js:413]: I imagine an actual Boolean value would work here too? Should we use that instead?

**Review by qiyundai — CHANGES_REQUESTED** [review]

File: blocks/events-form/events-form.js (hunk 1 above)
Comment (yeibercano) [reply blocks/events-form/events-form.js:413]: Unfortunately, the DOM API does not understand boolean. `true` would not work

Comment (yeibercano) [reply blocks/events-form/events-form.js:413]: let me check something else. Make sure the cache is not affecting this

Comment (yeibercano) [reply blocks/events-form/events-form.js]: Seems my local overrides just keep acting up. Anyhow, I pushed the Boolean type. Seems to work.

**Review by qiyundai — APPROVED** [review]

**Pull request merged**